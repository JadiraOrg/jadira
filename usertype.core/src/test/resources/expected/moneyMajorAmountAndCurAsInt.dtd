<!ELEMENT dataset (
    MONEYMAJORAMOUNTANDCURASINT*)>

<!ELEMENT MONEYMAJORAMOUNTANDCURASINT EMPTY>
<!ATTLIST MONEYMAJORAMOUNTANDCURASINT
    ID CDATA #REQUIRED
    MY_CURRENCY CDATA #IMPLIED
    MY_AMOUNT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

