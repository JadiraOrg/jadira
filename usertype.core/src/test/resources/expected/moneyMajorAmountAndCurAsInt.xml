<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "moneyMajorAmountAndCurAsInt.dtd">
<dataset>
  <MONEYMAJORAMOUNTANDCURASINT ID="0" MY_CURRENCY="840" MY_AMOUNT="100" NAME="test_0"/>
  <MONEYMAJORAMOUNTANDCURASINT ID="1" MY_CURRENCY="840" MY_AMOUNT="100" NAME="test_1"/>
  <MONEYMAJORAMOUNTANDCURASINT ID="2" MY_CURRENCY="978" MY_AMOUNT="0" NAME="test_2"/>
  <MONEYMAJORAMOUNTANDCURASINT ID="3" MY_CURRENCY="978" MY_AMOUNT="0" NAME="test_3"/>
  <MONEYMAJORAMOUNTANDCURASINT ID="4" NAME="test_4"/>
</dataset>